lib/screens/wallet/wallet_scroll_view.dart

```diff
@@ -378,45 +229,109 @@ class WalletScrollViewState extends State<WalletScrollView> {
               },
             ),
           ),
-        SliverToBoxAdapter(
-          child: Container(
-            color: ThemeColors.uiBackgroundAlt.resolveFrom(context),
-            padding: const EdgeInsets.fromLTRB(10, 10, 10, 10),
-            child: const Text(
-              'Transactions',
-              style: TextStyle(
-                fontSize: 22,
-                fontWeight: FontWeight.bold,
-              ),
+        if (wallet != null &&
+            wallet.doubleBalance == 0.0 &&
+            transactions.isEmpty)
+          SliverFillRemaining(
```

Comment:
This is basically `flex: 1` like in React Native behavior. 

It will fill the available space and the items inside will be constricted.

Better to use `SliverToBoxAdapter` and `Container`